src/assets/TestFramework/DeploymentCollection.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+namespace Azure.ResourceManager.TestFramework
+{
+    /// <summary> A class to add extension methods to ResourceGroup. </summary>
+    public class DeploymentCollection
```

Comment:
This class has been renamed to `ArmDeploymentCollection`
https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resources/Azure.ResourceManager.Resources/src/Generated/ArmDeploymentCollection.cs